__tests__/data/airship_input.json

```diff
@@ -73,8 +85,9 @@
           "version": "1.0.0"
         },
         "traits": {
-          "name": true,
-          "age": false
+          "email": "testone@gmail.com",
+          "firstName": "test",
+          "lastName": "one"
         },
```

Comment:
Can we test one test case with empty traits?

Reply:
Ya, have tested it.